pkg/utils/httpserver/server.go

```diff
@@ -4,13 +4,20 @@ import (
 	"context"
 	"fmt"
 	"net/http"
+	"strings"
 
 	"go.uber.org/zap"
 )
 
 func StartServer(ctx context.Context, log *zap.Logger, svc string, port string, handler http.Handler) {
+	var addr string
+	if strings.Contains(port, ":") {
+		addr = port
+	} else {
+		addr = fmt.Sprintf(":%s", port)
+	}
```

Comment:
This ensures that if the port is passed as `host:port` like in the case when pprof is **enabled** (like `localhost:6060`) format then do not add extra `:` in the addr. The addr value in other cases is passes like `:port` which means that the server will listen and serve for all hosts at port equal to `port`.